guides/source/upgrading_ruby_on_rails.md

```diff
@@ -257,16 +257,18 @@ it.
 
 `debugger` is not supported by Ruby 2.2 which is required by Rails 5. Use `byebug` instead.
 
-### Use bin/rails for running tasks and tests
+### Use rails or bin/rails for running tasks and tests
```

Comment:
I think backticks are allowed in headings, so it seems fine to make this [just] ``` `rails` ```. No need to call out `bin/rails` here; while this is about using it more often, people have already long been using it for other subcommands. (Likewise the sentence below.)

If we don't already address the version-handling etc elsewhere, I'd say the command line guide is the place to spend a paragraph on that (including the equivalence to `bin/rails`, and the fact it relies on that file).